src/coreComponents/mainInterface/initialization.cpp

```diff
@@ -114,10 +114,10 @@ std::unique_ptr< CommandLineOptions > parseCommandLineOptions( int argc, char *
     { SCHEMA, 0, "s", "schema", Arg::nonEmpty, "\t-s, --schema, \t Name of the output schema" },
     { NONBLOCKING_MPI, 0, "b", "use-nonblocking", Arg::None, "\t-b, --use-nonblocking, \t Use non-blocking MPI communication" },
     { PROBLEMNAME, 0, "n", "name", Arg::nonEmpty, "\t-n, --name, \t Name of the problem, used for output" },
-    { SUPPRESS_PINNED, 0, "s", "suppress-pinned", Arg::None, "\t-s, --suppress-pinned \t Suppress usage of pinned memory for MPI communication buffers" },
+    { SUPPRESS_PINNED, 0, "s", "suppress-pinned", Arg::None, "\t-s, --suppress-pinned, \t Suppress usage of pinned memory for MPI communication buffers" },
     { OUTPUTDIR, 0, "o", "output", Arg::nonEmpty, "\t-o, --output, \t Directory to put the output files" },
-    { TIMERS, 0, "t", "timers", Arg::nonEmpty, "\t-t, --timers, \t String specifying the type of timer output." },
-    { SUPPRESS_MOVE_LOGGING, 0, "", "suppress-move-logging", Arg::None, "\t--suppress-move-logging \t Suppress logging of host-device data migration" },
+    { TIMERS, 0, "t", "timers", Arg::nonEmpty, "\t-t, --timers, \t String specifying the type of timer output" },
+    { DISPLAY_MOVE_LOGGING, 0, "", "display-move-logging", Arg::None, "\t--display-move-logging, \t Display logging of host-device data migration" },
```

Comment:
How about having a short option for this too? My vote is for `-M` (uppercase, I don't want to reserve lowercase `m` for something that's not a primary parameter):
```suggestion
    { DISPLAY_MOVE_LOGGING, 0, "M", "display-move-logging", Arg::None, "\t-M, --display-move-logging, \t Display logging of host-device data migration" },
```

Reply:
I'll go for `trace-data-migration`.